staging/src/k8s.io/apiserver/pkg/admission/configuration/mutating_webhook_manager.go

```diff
@@ -57,6 +61,13 @@ func NewMutatingWebhookConfigurationManager(f informers.SharedInformerFactory) g
 		DeleteFunc: func(_ interface{}) { manager.updateConfiguration() },
 	})
 
+	// setup /configz endpoint
+	if cz, err := configz.New("mutatingwebhookconfigurations"); err == nil {
+		manager.configz = cz
```

Comment:
Do we open up another API this way? Do we document the compatibility guarantees for that?

Reply:
No, this is one of the `/configz`  configurations, which is a map.